cli/src/stake.rs

```diff
@@ -590,14 +590,14 @@ impl StakeSubCommands for App<'_, '_> {
                         .value_name("STAKE_ACCOUNT_ADDRESS")
                         .required(true),
                     "Stake account from which to withdraw (or base of derived address if --seed \
-                     is used). "
+                     is used)."
                 ))
                 .arg(pubkey!(
                     Arg::with_name("destination_account_pubkey")
                         .index(2)
                         .value_name("RECIPIENT_ADDRESS")
                         .required(true),
-                    "Recipient of withdrawn SOL"
+                    "Recipient of withdrawn SOL."
```

Comment:
nit: In one above place, we use `stake` instead of `SOL`
https://github.com/solana-labs/solana/blob/07f38838ae82333c026955ac78cd8d94f76a4b98/cli/src/stake.rs#L318

Maybe use `stake` to make this one consistent with that, or use `SOL` to make that one consistent with this one